scripts/android-build.conf

```diff
@@ -10,15 +10,15 @@
 #       create seperate configurations for each NDK API level
 #       and architecture you want to support.
 WITH_JPEG=0
-WITH_OPENH264=0
```

Comment:
Please deactivate this again, the pr builder does not have the required dependenciese installed.

Reply:
Would it be possible to prepare the builder for it? The effort should be manageable.
The advantage would be to have regression tests for each other pull request

Reply:
Deactivated as discussed